AppController/djinn.rb

```diff
@@ -1702,11 +1714,12 @@ def job_start(secret)
     write_our_node_info()
     wait_for_nodes_to_finish_loading(@nodes)
 
-    # We flush the logs on a separate thread.
+    # We update the dashboard on a separate thread.
     Thread.new {
       loop {
+        Kernel.sleep(20)
```

Comment:
Not really: since now we have the synchronous writing to the log file, this was a way to uniform to the main duty cycle. I did added a constant for it, which will be used for those main loop cycles.